### PR TITLE
Add support for SHOW LISTS metrics

### PR DESCRIPTION
**Pull request by SuperQ**
Add SHOW LISTS query metrics from pgbouncer documentation[0].

[0]: https://www.pgbouncer.org/usage.html

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

**Comment by SuperQ** [comment]
Related to: https://github.com/prometheus/docs/pull/1677#issuecomment-646011157

File: collector.go [hunk 1]
```diff
@@ -101,6 +137,38 @@ func NewExporter(connectionString string, namespace string, logger log.Logger) *
 	}
 }
 
+// Query SHOW LISTS, which has a series of rows, not columns.
+func queryShowLists(ch chan<- prometheus.Metric, db *sql.DB, logger log.Logger) error {
+	rows, err := db.Query("SHOW LISTS;")
+	if err != nil {
+		return errors.New(fmt.Sprintln("Error running SHOW LISTS on database: ", err))
```
Comment (brian-brazil) [inline collector.go:144]: Errors should start lowercase, as they may be prefixed by more errors.

**Comment by brian-brazil** [comment]
:+1:

File: collector.go (hunk 1 above)
Comment (SuperQ) [reply collector.go]: Fixed these. Thanks.

**Pull request merged**